docs/backends.md

```diff
@@ -88,10 +99,27 @@ Content-Length: 129
 {"id":"ca219243-0dd4-4e6e-94ad-54fbf3dd8b32","name":"my dell server","description":"some description","address":"192.168.100.4"}
 ```
 
+## View Backend Information
```

Comment:
include information on what this json file is used for on cocos and explain the fields it contains